sass/includes/_related-resources.scss

```diff
@@ -18,38 +18,53 @@
   }
 
   &__list-item {
-    background-color: $color__dark;
+    background-color: $color__light-grey;
     margin-bottom: 1rem;
-    padding: 1rem;
-    border-left: 1.25rem solid $color__yellow_2;
+    padding: 1rem 1.5rem;
+    border-left: 0.25rem solid $color__yellow_2;
     text-align: left;
+
+    @media (min-width: map-get($grid-breakpoints, "md")) {
+        border-left-width: 0.3125rem;
+    }
   }
 
   &__link {
-    text-decoration: none;
     transition: none;
+    text-decoration-thickness: 1px;
 
     &:hover {
-      text-decoration: underline;
+      text-decoration: none;
       color: $color__white;
     }
 
-    &:focus {
-      outline: 0.313rem solid $color__white;
-      outline-offset: .25rem;
-    }
+    // &:focus {
+    //   outline: 0.313rem solid $color__white;
+    //   outline-offset: .25rem;
+    // }
   }
 
   &__link-heading {
     margin-top: 0;
-    font-size: 1.625rem;
+    font-family: $font__body;
+    font-weight: $font-weight-bold;
+    font-size: 1.125rem;
```

Comment:
Can we use the font-size mixin for font sizes so these tie in with the sizes defined in the styleguide? If it's too much of a faff I can go through and refactor fonts later on https://github.com/nationalarchives/ds-wagtail/blob/686a4a07acd1cecfc7a01ac63ddd548712c77db0/sass/includes/_mixins.scss#L35

Reply:
I'm going to keep this consistent with the other PR so will leave it for now. Can come back and add in the font mixin once we've got the sizing updated across the board 👍  just to save any inconsistencies or discrepancies between design/FE.